chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -820,6 +820,10 @@ def replug(slave)
       ::Kernel.system("ovs-vsctl add-port #{@nic} #{slave}")
     end
 
+    def ovs_forward_bpdu(forward)
+      ::Kernel.system("ovs-vsctl set Bridge #{@nic} other-config:forward-bpdu=#{forward}")
```

Comment:
I am not entirely sure how ovs work internally. But wouldn't it be better to actually check the current value before writing it?
E.g. call `ovs-vsctl get Bridge #{@nic} other-config:forward-bpdu` first and only run the set command if an update is needed?

Reply:
s/other-config/other_config/ - `other-config` actually works too, but `other_config` is more in keeping with the convention.